tests/services/test_file_reconciliation.py

```
@@ -0,0 +1,128 @@
+        expected_symlink = hot_dir / "test.txt"
+        assert not expected_symlink.exists()
+        
+        stats = FileReconciliation.reconcile_missing_symlinks(path, db_session)
+        
+        assert stats["symlinks_created"] == 1
+        assert expected_symlink.is_symlink()
+        assert expected_symlink.resolve() == cold_file
+
+    def test_reconcile_symlinks_skip_non_symlink_path(self, db_session, monitored_path_factory):
+        """Test skipping reconciliation if path is not using SYMLINK operation."""
+        path = monitored_path_factory("Move Path", "/tmp/hot")
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/hot"

(S108)